### ISSUE TITLE
can't delete classroom

### ISSUE THREAD

**Issue by wschentw** [education/classroom#2570]
**Describe the bug**
the class room can't delete
I had type the classroom name, but the cancel button still no active

![image](https://user-images.githubusercontent.com/60765285/75610292-f7200600-5b4a-11ea-8d7a-7e45db61a92f.png)

**Comment by metinsenturk** [comment]
I have the same issue. Mine might be related to organization renaming. 
I've recently renamed my organization, and I had some demo classroom's that I made with my old organization name. 

Both created when my organization name was `spu-bigdataanalytics`.

spu-bigdataanalytics-classroom-2 
spu-bigdataanalytics-classroom-1

I renamed the organization to `spu-bigdataanalytics-193`.

Now I can't delete them. I am not sure if this is the cause of it, but It will be better if I can delete them.
